### DISCUSSION TITLE
AI Library - Like gdxAI

### DISCUSSION THREAD

**Discussion by anonimo82** [godotengine/godot-proposals#4366] [Scripting]
Hello everyone.
I propose an AI library for Visual/GD Script, something generic like gdxAI (https://github.com/libgdx/gdx-ai/wiki) - it's a general purpose AI framework for games, I guess Godot users could benefit from it.

Thanks for any help.

**Comment by Zireael07** [comment]
Try GDQuest's https://github.com/GDQuest/godot-steering-ai-framework

**Reply by anonimo82** [reply]
Wow, thanks a lot!
Exactly what I was looking for

**Comment by Calinou** [comment]
See also https://github.com/godotengine/godot-proposals/issues/281.

**Comment by anonimo82** [comment]
Guys, Godot community is great!
Thanks a lot for these links
